plugin/path_login.go

```diff
@@ -505,11 +500,12 @@ func (b *GcpAuthBackend) pathGceLogin(ctx context.Context, req *logical.Request,
 		Alias: &logical.Alias{
 			Name: alias,
 		},
-		Metadata:    authMetadata(loginInfo, serviceAccount),
 		DisplayName: instance.Name,
 	}
-
 	role.PopulateTokenAuth(auth)
+	if err := conf.GCEAuthMetadata.PopulateDesiredMetadata(auth, authMetadata(loginInfo, serviceAccount)); err != nil {
+		b.Logger().Warn("unable to populate gce metadata due", "err", err.Error())
```

Comment:
Nit: suggest to remove `due` at the end of this error message so that the message is similar to the IAM auth error message above.

Reply:
Good eye! Will fix!

Reply:
https://github.com/hashicorp/vault-plugin-auth-gcp/pull/92/commits/0dcbae999030110ed011ac6f89ba6c2992def536